internal/openshift/controller.go

```diff
@@ -86,19 +86,20 @@ func (c *controllerImpl) HandleBuild(o model.Object) error {
 		return nil
 	}
 
+	var sendUserToIdler bool
```

Comment:
IMHO it is better to be explicit about the initial state of the variable rather than relying on 0 value . IOW, how about `sendUserToIdler := false` ?

Reply:
done